ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
     }
 
-    .yaml_content_diagnosisContent {
+    .yaml_content_diagnosisBody {
+      position: relative;
       width: 400px;
```

Comment:
The border-radius has been increased from `8px` to `12px`. Please verify if this change aligns with the intended design.